examples/moist_bf_bubble.py

```diff
@@ -70,7 +70,7 @@
 Vt = theta0.function_space()
 Vr = rho0.function_space()
 x = SpatialCoordinate(mesh)
-quadrature_degree = (5, 5)
+quadrature_degree = (4, 4)
```

Comment:
Out of curiosity, any particular reason for lowering the quadrature degree? Is it a cost issue?